modules/soe_paragraph_stories/soe_paragraph_stories.info.yml

```
@@ -0,0 +1,12 @@
+name: 'Stories Paragraph'
+description: 'A paragraph type to display a person to highlight'
+type: module
+core_version_requirement: ^8.8 || ^9
+package: Stanford
+dependencies:
+  - ds
+  - field
+  - paragraphs
+  - text
+  - soe_paragraphs

```

```suggestion
  - ds:ds
  - drupal:field
  - paragraphs:paragraphs
  - drupal:text
  - soe_paragraphs:soe_paragraphs
```

In D8/D9 the dependencies are namespace.